src/test/kotlin/org/jetbrains/plugins/template/MyUITest.kt

```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.plugins.template
+
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.fixtures.ComponentFixture
+import com.intellij.remoterobot.fixtures.ContainerFixture
+import com.intellij.remoterobot.search.locators.byXpath
+import org.junit.Test
+import java.awt.event.KeyEvent
+
+class MyUITest {
```

Comment:
UI tests should ideally be in 
a) separate test source root
b) or separate test module